### PR TITLE
Allow adherent to create an unlimited number of Citizen Projects

### PR DESCRIPTION
**Pull request by illusionOfParadise**
Adherent can create an unlimited number of Citizen Projects but only in the case when he has no Citizen Project with pending, pre-approved or pre-refused status.

### CODE REVIEW

File: src/Repository/CitizenProjectRepository.php [hunk 1]
```diff
@@ -142,19 +142,15 @@ public function searchAll(SearchParametersFilter $search): iterable
         ;
     }
 
-    public function hasCitizenProjectInStatus(Adherent $adherent, array $status): bool
+    public function getCreatedCitizenProjectsNumber(Adherent $adherent): int
```
Comment (ottaviano) [inline src/Repository/CitizenProjectRepository.php]: what about `countCreatedC...`?
maybe `countCreatedCitizenProjectBy(Adherent $adherent): int`?

File: templates/citizen_project/edit.html.twig [hunk 2]
```diff
@@ -26,7 +26,13 @@
                 {{ include('components/caret--left.html.twig') }}
                 Retour au projet citoyen</a>
 
-            {% if not citizen_project.isApproved %}
+            {% if citizen_project.isRefused %}
+                <div class="committee__waiting-for-approval">
+                    En l'état, votre projet citoyen n'a pas été validé par nos équipes.
+                    Vous pouvez à tout moment modifier ce projet ou en proposer un nouveau (dans la limite de 3 projets par personne).
+                    Pour plus d’informations, reportez-vous à la Charte des Projets Citoyens <a class="link--blue" href="https://storage.googleapis.com/en-marche-prod/documents/projets-citoyens/La%20Charte%20des%20Projets%20Citoyens.pdf">la Charte</a>.
```
Comment (ottaviano) [inline templates/citizen_project/edit.html.twig]: are you sure for a position of the link? it's `à La Charte des Projects Citoyens La Charte.`

File: templates/citizen_project/show_base.html.twig [hunk 3]
```diff
@@ -62,9 +62,17 @@
...
     {% endif %}
 
-    {% if not citizen_project.isApproved %}
+    {% if citizen_project.isRefused %}
         <section>
-            <div class="citizen-project__waiting-for-approval flex--left flex--center--mobile">
+            <div class="citizen-project__waiting-for-approval flex--center--mobile">
+                En l'état, votre projet citoyen n'a pas été validé par nos équipes.
+                Vous pouvez à tout moment modifier ce projet ou en proposer un nouveau (dans la limite de 3 projets par personne).
+                Pour plus d’informations, reportez-vous à la Charte des Projets Citoyens <a class="link--blue" href="https://storage.googleapis.com/en-marche-prod/documents/projets-citoyens/La%20Charte%20des%20Projets%20Citoyens.pdf">la Charte</a>.
```
Comment (ottaviano) [inline templates/citizen_project/show_base.html.twig]: same for link

File: src/Security/Voter/CitizenProject/CreateCitizenProjectVoter.php [hunk 4]
```diff
@@ -27,10 +26,9 @@ protected function supports($attribute, $subject)
 
     protected function doVoteOnAttribute(string $attribute, Adherent $adherent, $subject): bool
     {
-        // Cannot create a project when already administrate one
+        // Cannot create a project when already has three citizen projects
         return $adherent->isAdherent()
-            && !$adherent->isCitizenProjectAdministrator()
```
Comment (ottaviano) [inline src/Security/Voter/CitizenProject/CreateCitizenProjectVoter.php]: are you sure for this deletion?

Comment (illusionOfParadise) [reply src/Security/Voter/CitizenProject/CreateCitizenProjectVoter.php]: Yes, adherent can create/have max 3 CP with any status, and so he can be an administrator of several CP.

**Pull request merged**